__tests__/app.test.js

```diff
@@ -39,18 +40,19 @@ describe("GET /api/topics", () => {
   });
 });
 
-describe("GET /api/topics", () => {
-  test("200: Responds with an array of all topic objects", () => {
+describe("GET /api/users", () => {
```

Comment:
did you remove your topics test and replace this with users? From the commit this is what it seems like so make sure you still have your /topics test!